content/en/events/2024/kcsna/social.md

```diff
@@ -14,14 +13,19 @@ There will be food, live music, activities, and more!
 
 ### Location
 
-To be announced
+[Flanker](https://www.google.com/maps/place/Flanker+Kitchen+%2B+Sporting+Club/@40.7696438,-111.9062182,17z/data=!3m2!4b1!5s0x8752f50009f98351:0x7edf0a3ab55fb2ac!4m6!3m5!1s0x8752f56087342337:0xdf17b52e1eccac58!8m2!3d40.7696398!4d-111.9036433!16s%2Fg%2F11n_nt0vqb?entry=tts&g_ep=EgoyMDI0MTAwMi4xIPu8ASoASAFQAw%3D%3D)
+
+6 N Rio Grande St Suite 35
```

Comment:
Did you want to put explicit line breaks (`<br>`) here too? Rendering makes this one line, which I think looks fine. But I see you separated them above, so just thinking about consistency.